package/Dockerfile

```diff
@@ -1,34 +1,39 @@
-FROM nginx:1.21.6-alpine
+FROM nginx:1.24.0-alpine
+
+ENV DOCKER_VERSION=23.0.3
```

Comment:
Was this tested to be compatible with the oldest Docker we support?

Reply:
I am not completely sure what version we do support, but generally that is defined via `DOCKER_API_VERSION`, which we are not setting in this project apart from the `entrypoint.sh`. What API version is this project meant to support?

@kinarashah can you please help testing this?

Reply:
You can override the rke-tools image to test using the `system_images` option in cluster.yml (CLI only)

Basically comes down to API support for the versions we do support (see support matrix)

Reply:
Aligned this with support matrix, and bumped to latest patch: `20.10.25`.

Reply:
I'm fine with `23.0.3` as long as its API compatible with the versions we support. But in the PR I am viewing it still says `23.0.3` while you say you bumped it to `20.10.25`

Reply:
Nice spot, that is now fixed as well.

Reply:
I kept the docker within the `20.x` series as that is aligned with other projects and our support matrix. 

The choice of `20.10.24` over `20.10.25`, was due to although the project has a [tag for `20.10.25`](https://github.com/moby/moby/releases/tag/v20.10.25), it does not have the artefacts published for them.